### PR TITLE
Video and Category listing

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Video and Category listing via API calls
- Context and Reducer implementation

### CODE REVIEW

File: src/components/Categories/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      {category.map(({_id, categoryName}) => (
+        <button key={_id} className="text-regular chips m-sm p-sm">{categoryName}</button>
+      ))}
+
+      {/* <button className="text-regular chips m-sm p-sm">
+                Training
+            </button>
+            <button className="text-regular chips m-sm p-sm">
+                Adoption
+            </button> */}
```
Comment (omjadhav85) [inline src/components/Categories/Categories.jsx:18]: can remove comments if not needed

File: src/context/data-context.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      try {
+        const {
+          data: { categories },
+        } = await axios.get("/api/categories");
+        dispatch({ type: "SET_CATEGORY", payload: [...categories] });
+      } catch (error) {
+        console.log(error);
+      }
+    })();
+  }, []);
```
Comment (omjadhav85) [inline src/context/data-context.js:35]: can be done in a single useEffect

**Review by omjadhav85 — APPROVED** [review]
Great work! Nice UI design and loved the use of css variables throughout the stylings. Can implement suggested minor changes else its good to merge!

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,26 @@
 import "./App.css";
-import { Aside } from "./components/Aside/Aside";
 import { Navbar } from "./components/Navbar/Navbar";
+import { Home } from "./pages/Home/Home";
+import { Routes, Route } from "react-router-dom";
+import { Watchlist } from "./pages/Watch Later/Watchlist";
+import { Playlist } from "./pages/Playlist/Playlist";
+import { History } from "./pages/History/History";
+import { LikedVideos } from "./pages/Liked Videos/LikedVideos";
```
Comment (prachi-sahani) [inline src/App.js:8]: you can use barrel imports

**Review by prachi-sahani — COMMENTED** [review]
Code is well structured.
All the changes are good to merge.

**Pull request merged**